sdk/python/feast/infra/aws.py

```diff
@@ -129,6 +129,7 @@ def get_historical_features(
         entity_df: Union[pandas.DataFrame, str],
         registry: Registry,
         project: str,
+        full_feature_names: bool = False,
```

Comment:
Are defaults (`False`) necessary here?

Reply:
I think just setting this default in `feature_store.py` may be sufficient. Will remove these default assignments from `aws.py`, `gcp.py`, `local.py` and `provider.py`.